app/controllers/idv/verify_info_controller.rb

```diff
@@ -32,56 +32,15 @@ def show
       process_async_state(load_async_state)
     end
 
-    def update
-      return if idv_session.verify_info_step_document_capture_session_uuid
-      analytics.idv_doc_auth_verify_submitted(**analytics_arguments)
-      Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
-        call('verify', :update, true)
+    private
 
+    def modify_pii_for_update
```

Comment:
I wonder if there's a more informative name for this method, or if it's really two methods, since the two controllers are doing such different things in it. What about:
```
update_state_id_type # remote controller does nothing in its method
update_uuid_prefix # in person controller does nothing
```
I'm surprised the in person controller doesn't need a uuid_prefix.

Reply:
Oh, now I see that. If they both set uuid_prefix the same way, you could leave that in the common update method and just have `update_state_id_type` which does different things in each controller, or maybe  `update_state_id_type if in_person`.